### DISCUSSION TITLE
First and Second Cover Formatting - NOVA IMS

### DISCUSSION THREAD

**Discussion by LaraNeves** [joaomlourenco/novathesis#234] [Q&A] [answered]
Dear Professor, 

First of all I would like to thank you for providing us with this template as it's been life saving for me, and I'm sure, for many other students. I'm using the latest version available on overleaf (I believe you released it this month). 

I found a few issues on the first cover of the NOVA IMS template, namely that:
1. "Dissertation submitted in ..." Has some gramatical errors and some formatting issues in that it repeats "in Master" twice. 
2. The subtitle is not showing up the way it's defined in the schl_defaults.ldf and the cover config file.
For both these issues I ended up hard-coding what I wanted there but I thought it would be somewhat useful to report in case there's a more elegant solution. 

Regardless, the main issue I'm having is with the second cover, where the advisers, the university and other types of information are not showing up. In addition to the text not being centred, as you can see here:

![image](https://user-images.githubusercontent.com/55484583/137649897-8c7c3fa8-045c-45ce-add1-038ef12a2aec.png)

My school requires me to submit something like this:
[second cover.pdf](https://github.com/joaomlourenco/novathesis/files/7361118/second.cover.pdf)

So my question is if you could point me to where I could alter the second cover to make it look more like official format. I would like to keep changes to the structural code to a minimum which is why I thought it would be wiser to ask you first in case you have a quick solution to my problem.

Thank you in advance for your time, I very much appreciate it.
Lara Neves

**Comment by joaomlourenco** [comment]
Hi Lara,

Thank you for your kind words.  Points 1 and 2 above were addressed and I think they are ok now.  Still, the default cover is not for your master.  May I ask you to please send me by email or upload here a copy of your cover for the Master in Advanced Analytics? Then I'll fix the second cover issue as well.  

Thanks,

**Reply by LaraNeves** [reply]
Many thanks for the quick response. Here's the cover template as provided by NOVA IMS:
[Templatethesis_MAA_EN.pdf](https://github.com/joaomlourenco/novathesis/files/7403117/Templatethesis_MAA_EN.pdf)
[Templatethesis_MAA_EN.docx](https://github.com/joaomlourenco/novathesis/files/7403271/Templatethesis_MAA_EN.docx)

And I'll add the Portuguese version as well:

[Template_Tese_MAA_PT.pdf](https://github.com/joaomlourenco/novathesis/files/7403119/Template_Tese_MAA_PT.pdf)
[Template_Tese_MAA_PT.docx](https://github.com/joaomlourenco/novathesis/files/7403272/Template_Tese_MAA_PT.docx)

Please let me know if you need anything else. Thank you!

**Reply by b-pereira** [reply]
@LaraNeves I'm cleaning up the Unanswered questions in the Q&A. I asked @joaomlourenco about this Question, and he mentioned that he's terribly occupied and only in the end of this month (November/2021) is when is schedule has some availability. **So, the question is how much urgent is this for you?**

**Reply by LaraNeves** [reply]
Hey @b-pereira I need to deliver the final version of my thesis by the 20th of November, so I'm quite pressed for time to be honest. I understand that @joaomlourenco is busy and it wouldn't be fair to press him with this. So just to clarify, is there anyway to have those covers ready by then or should I find some way to solve the issue? 

Many thanks for keeping in touch. Really appreciate it!

**Reply by b-pereira** [reply]
Let's see how @joaomlourenco replies to this. Meanwhile, I'll try to talk with him. It's almost November 20th, so even if he could delegate that to me, I don't know if something can complicate until then and I couldn't meet that due date. Don't worry!

**Comment by joaomlourenco** [comment]
Please check the latest version (6.6.2). I hope it is ok.

You need to:

1. Make a backup of your working document (Murphy is around).
1. Download the new version.
2. Overwrite the new "Config" folder with your current "Config" folder.
3. Add `\ntsetup{degree=mmaa}` to your "Config/1_novathesis.tex" file (see the new original file before overwriting).
5. Overwrite the new "Chapters" folder with your current "Chapters" folder.
6. Overwrite the new "Bibliography" folder with your current "Bibliography" folder.

Hopefully it will work.

**Reply by joaomlourenco** [reply]
Lara, please let me know ASAP if it works for you, or if something else is needed!

**Reply by b-pereira** [reply]
@joaomlourenco it's better to use @LaraNeves handle. Here's why — [Mention @somebody. They’re notified](https://github.blog/2011-03-23-mention-somebody-they-re-notified/). To be honest, I really don't know to what degree someone gets a notification if they created the issue/discussion. But with the @somebody for sure they are notified.

I'm watching the repository, I always get ANY notifications in Github's Android app and in the email. What's in the image is what I see from my side:
![imagem](https://user-images.githubusercontent.com/944511/141819088-24092541-5e06-4616-8466-987b0c10560d.png)

**Reply by LaraNeves** [reply]
@joaomlourenco Apologies for the wait, yes! The first and second cover are working perfectly, many many thanks!! Really appreciate it. As for what @b-pereira said, I don't get much notifications despite having created the discussion, only when I'm mentioned which is why it takes me a while to answer at times. Again many thanks! 

Just a quick note on the side cover, it doesn't match perfectly with the MMAA format as is at the moment. Because it's not a big issue for me as I can fix that easily enough, I won't insist on it as I know you guys are very busy, but I'll just mention it here for future developments in case other students from NOVA IMS in the MMAA masters need to use the template. 

Again many thanks, you guys are doing an amazing job and I will be sure to give you donation as soon as I can, because you saved me the trouble of having to use Word which, as we all know, would only double the PAIN of having to write a thesis.

Hope you'll have a fantastic week!
Lara

**Reply by joaomlourenco** [reply]
@LaraNeves, please clarify what's the problem with the side cover?  Thanks.

**Reply by LaraNeves** [reply]
The current side cover I have using the latest template:
[current_side_cover.pdf](https://github.com/joaomlourenco/novathesis/files/7550402/current_side_cover.pdf)
The expected side cover as required by NOVA IMS:
[required_side_cover.pdf](https://github.com/joaomlourenco/novathesis/files/7550423/required_side_cover.pdf)

**Comment by joaomlourenco** [comment, marked as answer]
@LaraNeves, vai ao repositório buscar os ficheiros
```
 NOVAthesisFiles/Schools/nova/ims/schl-defaults.ldf
 NOVAthesisFiles/Schools/nova/ims/spine.tex
```
e substitui os que tens no teu template.  

A lombada ficará de acordo com as normas (o texto do título e nome do autor são redimensionados automaticamente para caberem nas respetivas caixa brancas).

Por favor confirma que está ok para ti!  Obrigado.  Se estiver, marca esta discussão como "answered".  Obrigado.
![image](https://user-images.githubusercontent.com/2064643/142218663-2abe012e-7472-47c3-8142-b1b4e0184184.png)

**Reply by LaraNeves** [reply]
@joaomlourenco Substituir apenas esses dois ficheiros não foi o suficiente, tive que criar um projecto de novo com a última versão do template e depois copiar os meus ficheiros. Mas funcionou lindamente! A primeira, segunda capa e lombada estão perfeitas. Mil obrigadas! O professor poupou-me dias de trabalho e uma grande dor de cabeça, não tenho como agradecer o suficiente. Muito obrigada mesmo!

**Comment by joaomlourenco** [comment]
@LaraNeves, uma boa forma de “agradeceres” é divulgares o template nos fóruns dos alunos de mestrado e doutoramento da IMS.   Era fixe ter mais utilizadores na IMS. 
Sugiro que publiques uma mensagem acompanhada de uma seleção de páginas (capa, folha de rosto, lombada, um par de páginas do texto com uma figura ou tabela, etc), com um link para o repositório ou assim…
Obrigado também.  Boa sorte para as provas.
